Library/ViewModels/ThanksCategoryCellViewModel.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol ThanksCategoryCellViewModelInputs {
+  func allProjectCategoryButtonTapped()
+  func configureWith(category: KsApi.Category)
+}
+
+public protocol ThanksCategoryCellViewModelOutputs {
+  var notifyToGoToDiscovery: Signal<KsApi.Category, Never> { get }
```

Comment:
I wonder if this could be `notifyDelegateGoToDiscovery` so we're specific about what object is being notified.